### PR TITLE
Add duplicate of description field for spanish translations

### PR DESCRIPTION
**Pull request by zainab-amir**
Created new field for school pages in Disco to allow for alternate translated text to be manually inputted.

- Updated model to include `description_es` field
- Updated serializer to include `description_es`
-----

Locally `/api/v1/organizations/` now includes the newly added field in it's response:

Screenshot 2021-03-19 at 5 16 15 PM

[DISCO-1737](https://openedx.atlassian.net/browse/DISCO-1737)

### CODE REVIEW

**Comment by zainab-amir** [comment]
Removed `null=True` from the `description_es` field, details:
```
     = Avoid using null on string-based fields such as CharField and TextField.
       If a string-based field has null=True, that means it has two possible
       values for "no data": NULL, and the empty string. In most cases, it's
       redundant to have two possible values for "no data;" the Django
       convention is to use the empty string, not NULL.
```

File: course_discovery/apps/course_metadata/models.py [hunk 1]
```diff
@@ -168,6 +168,7 @@ class Organization(CachedMixin, TimeStampedModel):
     )
     slug = AutoSlugField(populate_from='key', editable=False, slugify_function=uslugify)
     description = models.TextField(null=True, blank=True)
+    description_es = models.TextField(blank=True)
```
Comment (waheedahmed) [inline course_discovery/apps/course_metadata/models.py]: You might need to set `null=True` as well.

Comment (zainab-amir) [reply course_discovery/apps/course_metadata/models.py]: If I set `null=True` the CI / semgrep check fails with the above error

Comment (zainab-amir) [reply course_discovery/apps/course_metadata/models.py]: I think we can keep it this way and inform website squad that the default value will be empty string

Comment (waheedahmed) [reply course_discovery/apps/course_metadata/models.py]: Got it.

Comment (waheedahmed) [reply course_discovery/apps/course_metadata/models.py]: Specify `verbose_name=_('Spanish Description')` and a help text specifying the purpose of the field.

**Review by waheedahmed — APPROVED** [review]
A couple of suggestions and don't forget to recreate the migration after these changes.

**Review by waheedahmed — APPROVED** [review]

**Pull request merged**